packages/lab/src/divider/Divider.css

```diff
@@ -0,0 +1,21 @@
+.saltDivider-horizontal {
+  border-bottom: var(--salt-size-border) var(--salt-separable-borderStyle);
+  width: 100%;
+}
+
+.saltDivider-primary {
+  border-color: var(--salt-separable-primary-borderColor);
+}
+
+.saltDivider-secondary {
+  border-color: var(--salt-separable-secondary-borderColor);
+}
+
+.saltDivider-tertiary {
+  border-color: var(--salt-separable-tertiary-borderColor);
+}
+
+.saltDivider-vertical {
```

Comment:
This should be placed above the variant classes

Reply:
done